dmoj/monitor.py

```diff
@@ -52,11 +52,20 @@ def run(self):
 
 
 class SendProblemsHandler(FileSystemEventHandler):
+    ALLOWED_EVENT_TYPES = (
+        'modified',
```

Comment:
Can you use the named constants https://github.com/gorakhargosh/watchdog/blob/f3dfc4be0c6ec36c2db6b4afa075004e71adf87e/src/watchdog/events.py#L103-L108 instead of hardcoding the strings?